cdap-docs/developers-manual/source/building-blocks/index.rst

```diff
@@ -48,6 +49,11 @@ write data through the data abstraction layer in CDAP.
 - :doc:`Workers <workers>`
 - :doc:`Workflows <workflows>`
 
+:doc:`Metadata <metadata>` |---| consisting of **properties** (a list of key-value pairs)
+or **tags** (a list of keys) |---| can be set for both data and applications. These can be
+retrieved and searched, and the metadata is used to track and view the lineage of datasets
```

Comment:
Metadata is not used to track lineage. Accesses are used to track lineage. Metadata is used for discovery of CDAP entities.